src/main/java/network/brightspots/rcv/RawContestConfig.java

```diff
@@ -46,11 +46,11 @@
   public static class OutputSettings {
 
     public String contestName;
+    public String contestId;
```

Comment:
I'm kind of baffled why this field was sitting here all along, and just unused / not hooked up to the GUI before. Any historical context on this?

Reply:
@moldover just added this in #458.